jib-maven-plugin/build.gradle

```diff
@@ -11,6 +11,7 @@ dependencies {
   sourceProject project(':jib-build-plan')
   sourceProject project(':jib-core')
   sourceProject project(':jib-plugins-common')
+  sourceProject project(':jib-maven-plugin-extension-api')
```

Comment:
this one too, I'm unsure of, should we depend on what's publicly available?

Reply:
Hmm... yeah, I think we should use a published one. But haven't someone said if we don't make a breaking change in `jib-maven-plugin-extension-api`, then including source will be acceptable?

How should we proceed at the moment?